apps/todoistnext/todoistnext.go

```diff
@@ -0,0 +1,25 @@
+// Package todoistnext provides details for the Todoist Next applet.
+package todoistnext
+
+import (
+	_ "embed"
+
+	"tidbyt.dev/community/apps/manifest"
+)
+
+//go:embed todoist_next.star
+var source []byte
+
+// New creates a new instance of the Todoist Next applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "todoist-next",
+		Name:        "Todoist Next",
+		Author:      "Alisdair/Akeslo",
```

Comment:
Did you hear back from the original author? Not sure if we really can merge it without their OK.

Reply:
hmm ok, I did not hear back. I'll ping again and if no response I can just rewrite and include caching.

Reply:
Hi, I am the original author. Please go ahead and well done!

Reply:
Sweet!